epub33/core/index.html

```diff
@@ -4417,13 +4417,14 @@ <h4>Container-Constrained Scripts</h4>
 				<section id="sec-scripted-spine">
 					<h4>Spine-Level Scripts</h4>
 
-					<p id="confreq-cd-scripted-spine">EPUB Content Documents that include <a
-							href="#sec-scripted-content-type-spine-level">spine-level scripting</a> MUST utilize the
-							<em>progressive enhancement technique</em>, which for the purposes of this specification has
-						the following definition: when the document is rendered by a Reading System without scripting
-						support or with scripting support disabled, the <a>Top-level Content Document</a> MUST retain
-						its integrity, remaining consumable by the user without any information loss or other
-						significant deterioration.</p>
+					<p id="confreq-cd-scripted-spine"><a>Top-level Content Documents</a> that include <a
+							href="#sec-scripted-content-type-spine-level">spine-level scripting</a> SHOULD remain
+						consumable by the user without any information loss or other significant deterioration when
+						scripting is disabled (e.g., by employing progressive enhancement techniques).</p>
+
```

Comment:
I do not feel strongly about this, but we may consider adding an (informative) reference to https://developer.mozilla.org/en-US/docs/Glossary/Progressive_Enhancement.

I leave to you to decide whether that is necessary.

Reply:
How would we create a cite for it? An informative "MDN" entry in the local biblio that just states the site name?

Or does W3C allow hyperlinking to external sources without citations?

Reply:
Good point. I have spent some time getting around existing documents, but hyperlinking to external sources does occur but not in a normative section (and this one is). We can either drop the idea, or put the reference into a separate (small) note. But that may not worth it...

Reply:
I think we're safe leaving it as is. As @dauwhe says, it's a widely known term and searching returns only relevant results, amusingly including the same definition in adjoining boxes:

<img width="821" alt="wikipedia definitions" src="https://user-images.githubusercontent.com/1565164/104481071-b468f000-559b-11eb-95cc-8305b7aeff10.png">

Reply:
O.k. Go for it!